lib/model/migrations/20220628-01-enable-pg-stat-statements.js

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = (db) => db.raw('create extension if not exists pg_stat_statements');
```

Comment:
Similar migrations seem to call `db.schema.raw()`:

https://github.com/getodk/central-backend/blob/4f9d19c1e5a604385a36c5a3901e1d8d0f3c4bda/lib/model/migrations/20181011-make-email-case-insensitive.js#L10-L11
https://github.com/getodk/central-backend/blob/4f9d19c1e5a604385a36c5a3901e1d8d0f3c4bda/lib/model/migrations/20190225-01-add-actor-trigram-indices.js#L10-L11


Is that needed here? Is there a difference here between `db.raw()` and `db.schema.raw()`?